### DISCUSSION TITLE
Recover my loop in funds

### DISCUSSION THREAD

**Discussion by abcb1122** [Ride-The-Lightning/RTL#1371] [LND]
Hello I did a Loop In from an external wallet. Opened RTL did the loop in request. Copied the address but when I sent the funds the order changed to failed and no sats got to my channels, the funding tx is confirmed more than 10 blocks.
How can I recover my funds?

**Comment by saubyk** [comment]
Hi @abcb1122 external funding of loop-in is not a feature in RTL. Not sure why you made that assumption and did that transaction.
The question of a possible recovery of funds is not something we can address, you need to reach out to the loop team.

https://github.com/lightninglabs/loop

**Reply by abcb1122** [reply]
Ok Mr, thank you.
